bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieProtocol.java

```diff
@@ -180,6 +180,11 @@ public static short getFlags(int packetHeader) {
      */
     int ETOOMANYREQUESTS = 106;
 
+    /**
+     * Client could not be authenticated.
+     */
+    int EAUTH = 107;
```

Comment:
Why aren't you using existing code EUA (102)?
If we use EUA this change is compatible with old clients seamlessly